packages/toolkit/monorepo-utils/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@modern-js/monorepo-utils",
+  "description": "A Progressive React Framework for modern web development.",
+  "homepage": "https://modernjs.dev",
+  "bugs": "https://github.com/web-infra-dev/modern.js/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/web-infra-dev/modern.js",
+    "directory": "packages/toolkit/monorepo-utils"
+  },
+  "license": "MIT",
+  "keywords": [
+    "react",
+    "framework",
+    "modern",
+    "modern.js"
+  ],
+  "version": "2.22.0",
+  "jsnext:source": "./src/index.ts",
+  "types": "./src/index.ts",
+  "main": "./dist/cjs/index.js",
+  "exports": {
+    ".": {
+      "require": "./dist/cjs/index.js",
+      "import": "./dist/esm/index.js",
+      "default": "./dist/cjs/index.js"
+    }
+  },
+  "_comment": "Provide ESM and CJS exports, ESM is used by runtime package, for treeshaking",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public",
+    "types": "./dist/types/index.d.ts"
+  },
+  "scripts": {
+    "prepublishOnly": "only-allow-pnpm",
+    "new": "modern-lib new",
+    "dev": "modern-lib build --watch",
+    "build": "modern-lib build",
+    "test": "jest --passWithNoTests"
+  },
+  "dependencies": {
+    "@microsoft/rush-lib": "^5.99.0",
```

Comment:
`@microsoft/rush-lib` contains so many sub-dependencies, maybe we should move it to devDependencies.

Reply:
With these considerations in mind, perhaps we should use builder-plugin-build-source as an external plugin.

Reply:
`@microsoft/rush-lib` has been removed from the project.